tests/test_helpers/dom_helper.ts

```diff
@@ -151,3 +152,23 @@ export function getElComputedStyle(selector: string, style: string): string {
   if (!element) throw new Error(`No element matching selector "${selector}"`);
   return window.getComputedStyle(element)[style];
 }
+
+export async function dragElement(
+  element: Element,
+  dragX: Pixel,
+  dragY: Pixel,
+  dontMouseUp = false
```

Comment:
Double negations are hard to read (`!dontMouseUp`)
```suggestion
  mouseUp = false
```
And I propose to introduce
```ts
export async function dragElementAndRelease(...) {
  await dragElement(..., true)
}
```